specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/applicationGateway.json

```diff
@@ -2076,6 +2430,148 @@
       ],
       "description": "Redirect configuration of an application gateway."
     },
+    "ApplicationGatewayPrivateLinkResourcePropertiesFormat": {
+      "properties": {
+        "subnet": {
```

Comment:
>"subnet" [](start = 8, length = 8)

the privateLinkResources contract only includes groupId, requiredZoneNames, and requiredMembers.  Why is this deviating from that contract?